tests/test_catalog_and_language.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from figo import FigoException
+from figo import FigoSession
+from figo.models import Service
+from figo.models import LoginSettings
+
+CREDENTIALS = ["figo", "figo"]
+BANK_CODE = "90090042"
+CLIENT_ERROR = 1000
+
+@pytest.mark.parametrize('language', ['de', 'en'])
+def test_get_catalog_en(access_token, language):
+    figo_session = FigoSession(access_token)
```

Comment:
You can create a `session` fixture to remove the session creation from the real tests.

Reply:
No, I am creating sessions per tests, because these tests fiddle around with language header - and that's adding state to tests. Since I don't want to use the fixture with user creation (overhead), I settled with one user, but multiple sessions, one per test. Also note, that you can not use a fixture with function scope from a fixture with session scope (and vice versa).